src/js/stores/CosmosPackagesStore.js

```diff
@@ -87,7 +91,10 @@ class CosmosPackagesStore extends GetSetBaseStore {
         repositoryAddSuccess: COSMOS_REPOSITORY_ADD_SUCCESS,
         repositoryAddError: COSMOS_REPOSITORY_ADD_ERROR,
         repositoryDeleteSuccess: COSMOS_REPOSITORY_DELETE_SUCCESS,
-        repositoryDeleteError: COSMOS_REPOSITORY_DELETE_ERROR
+        repositoryDeleteError: COSMOS_REPOSITORY_DELETE_ERROR,
+
+        serviceDescriptionSuccess: COSMOS_SERVICE_DESCRIBE_CHANGE,
```

Comment:
not big deal but based on the other constants shouldn't this be `COSMOS_SERVICE_DESCRIBE_SUCCESS`?

Reply:
`COSMOS_SERVICE_DESCRIBE_CHANGE` follows the same naming as the existing `COSMOS_DESCRIBE_CHANGE`. These are "sister" endpoints, so I think current naming makes sense.

Reply:
Judging from this file, we seem to use "success" when an action was performed that affects the cluster, and we use "change" when fetching some kind of state.